src/main/java/uk/gov/pay/adminusers/persistence/entity/StripeAgreementEntity.java

```diff
@@ -0,0 +1,56 @@
+package uk.gov.pay.adminusers.persistence.entity;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+import java.time.LocalDateTime;
+
+@Entity
+@Table(name = "stripe_agreements")
+@SequenceGenerator(name = "stripe_agreements_id_seq_gen", sequenceName = "stripe_agreements_id_seq", allocationSize = 1)
+
+public class StripeAgreementEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "stripe_agreements_id_seq_gen")
+    private int id;
+
+    @Column(name = "service_id")
+    private int serviceId;
+
+    @Column(name = "ip_address")
+    private String ipAddress;
+
+    @Column(name = "agreement_time", columnDefinition = "TIMESTAMP WITHOUT TIME ZONE NOT NULL")
+    private LocalDateTime agreementTime;
```

Comment:
I guess when we set the value for this property we are going to use `UTC`, for example:
```
LocalDateTime.now(ZoneOffset.UTC)
```